driver/postgres/postgres.go

```diff
@@ -59,8 +60,14 @@ func (driver *Driver) ensureConnectionNotClosed() error {
 	return nil
 }
 
-func (driver *Driver) Close() error {
-	if err := driver.db.Close(); err != nil {
+func (p *Driver) Close() error {
```

Comment:
1. we use generic.go driver also, not only postgres driver (driver/generic/generic.go)
so we should handle the Close() error there also
2. the Error constant name and string is not accurate - the error is not that the connection is closed, the error is that failed to send closeNotify alert
3. The "Contains" expression looks too specific - maybe we could do without the  ": write tcp" suffix